Formula/haskell-stack.rb

```diff
@@ -5,8 +5,9 @@ class HaskellStack < Formula
 
   desc "The Haskell Tool Stack"
   homepage "https://haskellstack.org/"
-  url "https://github.com/commercialhaskell/stack/archive/v2.3.1.tar.gz"
-  sha256 "6701ddfc6d0be0c2bf0f75c84375e41923c5617f04222c5e582e7011c7f8fb83"
+  url "https://github.com/commercialhaskell/stack/releases/download/v2.3.3/stack-2.3.3-osx-x86_64.tar.gz"
```

Comment:
This doesn't look like a correct url.

Reply:
hmm, that is the one I actually used before. looks like the CI build failure might be because of this change.